cosmos/providers/dbt/core/operators.py

```diff
@@ -307,6 +307,22 @@ def execute(self, context: Context):
         result = self.build_and_run_cmd(env=self.get_env(context), cmd_flags=cmd_flags)
         return result.output
 
+class DbtDocsOperator(DbtBaseOperator):
```

Comment:
This operator allows `dbt-ol` to pull all possible info from a project, but it is not strictly necessary.